cli/src/commands/claim.ts

```diff
@@ -0,0 +1,28 @@
+import Command from '../utils/command';
+import terminalLink from 'terminal-link';
+
+export default class Claim extends Command {
+  public static description = 'claim an application';
+  public static hidden = false;
+  public static examples = [
+    `$ ${Command.cliBinName} claim OiJIUzI1NiIsIn`,
+  ];
+
+  public static args = [
+    {
+      name: 'token',
+      required: true,
+      description: 'claim token',
+    },
+  ];
+
+  public static strict = true;
+
+  public async run() {
+    const { args } = this.parse(Claim);
```

Comment:
You should check that you're authenticated here: https://github.com/binaris/shiftjs/blob/b76a1672cde2422afa48e346043584ab9f766ac9/cli/src/commands/deploy.ts#L116